sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -89,30 +89,33 @@ public Stream BodyAsStream
         /// </summary>
         ///
         /// <remarks>
-        ///   This property is only populated for events received from the Event Hubs service.
+        ///   This property is only populated for events received from the Event Hubs service. If this
+        ///   EventData was not recived from the Event Hubs service, the value is long.MinValue.
         /// </remarks>
         ///
-        public long? SequenceNumber { get; }
+        public long SequenceNumber { get; }
 
         /// <summary>
         ///   The offset of the event when it was received from the associated Event Hub partition.
         /// </summary>
         ///
         /// <remarks>
-        ///   This property is only populated for events received from the Event Hubs service.
+        ///   This property is only populated for events received from the Event Hubs service. If this
+        ///   EventData was not recived from the Event Hubs service, the value is long.MinValue.
         /// </remarks>
         ///
-        public long? Offset { get; }
+        public long Offset { get; }
 
         /// <summary>
         ///   The date and time, in UTC, of when the event was enqueued in the Event Hub partition.
         /// </summary>
         ///
         /// <remarks>
-        ///   This property is only populated for events received from the Event Hubs service.
+        ///   This property is only populated for events received from the Event Hubs service. If this
+        ///   EventData was not recived from the Event Hubs service, the value default(DateTimeOffset).
```

Comment:
nit:  Would you mind wrapping in code tags for consistency with other doc comments?   `<c>default(DateTimeOffset</c>`